mono/mini/interp/mintops.h

```diff
@@ -53,11 +53,18 @@ enum {
 
 #define MINT_SWITCH_LEN(n) (3 + (n) * 2)
 
-extern const char * const mono_interp_opname[];
 extern unsigned char const mono_interp_oplen[];
 extern MintOpArgType const mono_interp_opargtype[];
 extern char* mono_interp_dis_mintop(const unsigned short *base, const guint16 *ip);
 extern const guint16* mono_interp_dis_mintop_len (const guint16 *ip);
 
+// This, instead of an array of pointers, to optimize away a pointer and a relocation per string.
+extern const guint16 mono_interp_opname_offsets [ ];
+typedef struct MonoInterpOpnameCharacters MonoInterpOpnameCharacters;
+extern const MonoInterpOpnameCharacters mono_interp_opname_characters;
+
+#define mono_interp_opname(x) (((const char*)&mono_interp_opname_characters) + mono_interp_opname_offsets [x])
```

Comment:
Could you make this a normal global function ? It would be really annoying if I didn't have access to this while in debugger.